source/agora/common/crypto/ECC.d

```diff
@@ -425,3 +429,12 @@ unittest
     assert(points[0] == Point.fromString(
             "0x37e8a197247dd01cc27c178dc0465ce826b4f6e312f3ee4c1df0623ef38c51c5"));
 }
+
+unittest
+{
+    // import std.stdio : writefln;
+    // import std.conv : to;
+    // writefln("point= %s", to!string(Scalar.random().toPoint()));
+    auto p = Point.fromString("0xab4f6f6e85b8d0d38f5d5798a4bdc4dd444c8909c8a5389d3bb209a18610511b");
+    assert(p.isValid());
```

Comment:
It also needs an invalid test. For example `Point point. assert(!p.isValid());`

Reply:
Updated